src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -78,23 +78,31 @@ <h1 class="subheader__title">
                         {% if course.duration or course.effort or current_page.publisher_is_draft %}
                         <div class="characteristics">
                             <ul class="characteristics__container">
-                            {% render_model_block course "duration,effort" %}
+                            {% render_model_block course "duration,effort,is_self_paced" %}
                                 {% if instance.duration or current_page.publisher_is_draft %}
                                 <li class="characteristics__item">
                                     <svg role="img" class="characteristics__icon" aria-hidden="true">
-                                        <use href="#icon-clock" />
+                                        <use href="#icon-calendar" />
```

Comment:
An interval or a meter would be nice? To keep the calendar for dates. Something like:
![image](https://user-images.githubusercontent.com/1427165/109783462-50ad8b80-7c0a-11eb-95ac-51fe7fdce01e.png)
![image](https://user-images.githubusercontent.com/1427165/109783481-560ad600-7c0a-11eb-8626-daa58c9e46b8.png)

Reply:
Not a fan of these icons, we loose time context.

Reply:
For the ruler I agree. But the one with the arrows is quite explicit for time frame no?

Reply:
![Duration](https://user-images.githubusercontent.com/9265241/109800540-96c01a80-7c1d-11eb-8fa2-38d19dcad8ad.png)

I finally use this one. (Take a look to screenshots joined in description)

Reply:
Nice and smart! Maybe fine-tune vertical alignment with the text. Seems low.

Reply:
@sampaccoud  It should be better now! I also used a thinner clock icon to be aligned graphically.
It is ready for a new review.